File: C/URI 1011.c
#include <stdio.h>
int main(void) {
  double R, VOLUME;
  scanf("%lf",&R);
  VOLUME = (4/3.0) * 3.14159 * (R*R*R);
  printf("VOLUME = %.3lf\n",VOLUME);
  return 0;
}

File: Java/URI 1011.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      double raio, VOLUME;
      raio = sc.nextDouble();
      VOLUME = (4.0/3.0) * (3.14159) * (raio * raio * raio);
      System.out.printf("VOLUME = %.3f%n", VOLUME);
      sc.close();
  }
}